CONTRIBUTING.md

```diff
@@ -8,19 +8,19 @@ We use [RushJS](https://rushjs.io) to manage this repository. Some quick notes o
 
 **Install dependencies** -- You only need to run this once.
 
-```sh
+```
```

Comment:
😠

Reply:
Lost the semantics here. This is not just _any_ code, it's Borne Shell code!

Reply:
True but `sh` does not work in any editor. That's why I removed it ;).

Reply:
Here is the editor you want to be using for Markdown!

https://typora.io